Sources/DistributedActors/ClusterSystem.swift

```diff
@@ -993,6 +993,10 @@ extension ClusterSystem {
         Err: Error,
         Res: Codable
     {
+        if let interceptor = actor.id.context.remoteCallInterceptor {
```

Comment:
I wonder if we can move this inside `self.withCallID` so the interceptor doesn't need to worry about timeouts.

Reply:
Hmm I see, but it might have to do extra cleanup -- like this one does, when the stashed message times out, we'd like to drop it I think?